examples/dowelTest1.py

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
We usually use `pytest` to test our code. Can you move your tests to the `test/` directory and use pytest to automate them?

Reply:
Created test cases inside test_csv_ouput